src/App.js

```diff
@@ -58,8 +61,34 @@ const App = () => {
   };
 
   const startEditingContent = () => {
-    const contentEditTool = documentViewer.getTool(window.Core.Tools.ToolNames.CONTENT_EDIT);
-    documentViewer.setToolMode(contentEditTool);
+    const contentEditManager = documentViewer.getContentEditManager();
+    contentEditManager.startContentEditMode();
+    setIsInContentEditMode(true);
+  }
+
+  const endEditingContent = () => {
+    setIsInContentEditMode(false);
+    documentViewer.setToolMode(window.Core.Tools.ToolNames.EDIT)
```

Comment:
This should be the following. Right now it's not working.
```suggestion
    documentViewer.setToolMode(documentViewer.getTool(window.Core.Tools.ToolNames.EDIT));
```